generic/_generic.reset.scss

```diff
@@ -33,24 +76,19 @@ li > {
 }
 
 
-
 /**
- * Remove default table spacing.
+ * Prevent double borders.
  */
 
-table {
+ table {
```

Comment:
There is a extra space before the selector 👾

Reply:
Thank you